ui/src/app/utils/institutions.tsx

```diff
@@ -82,10 +84,18 @@ export function getRegisteredTierConfig(institution: Institution): InstitutionTi
   return getTierConfig(institution, AccessTierShortNames.Registered);
 }
 
+export function getControlledTierConfig(institution: Institution): InstitutionTierConfig {
```

Comment:
that is the initiali version in my previous PR, then I feel the code becomes harder to writer/read, and redudant because we make getRt/CTTierConfig/Address/Domains is used in so many places, and each time we have to pass the tier short name.

Reply:
fine for now - would like to look at this in our refactor pairing as well